### PR TITLE
Use the Django TestCase's Client

### PR DESCRIPTION
**Pull request by tnightingale**
Use the `django.test.Client` instance that Django provides with its `TestCase` class.

This allows GraphQL tests to make use of the stateful client methods like `client.login()`.

### CODE REVIEW

**Comment by ulgens** [comment]
ping @jkimbo

File: graphene_django/utils/testing.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import json
 
-from django.test import TestCase, Client
+from django.test import Client, TestCase
```
Comment (ulgens) [inline graphene_django/utils/testing.py:3]: After the change below, is that `Client` class used anywhere?

Comment (tnightingale) [reply graphene_django/utils/testing.py:3]:
```suggestion
from django.test import TestCase
```

**Comment by ulgens** [comment]
@tnightingale @zbyte64 Any change we rebase & merge this?

**Comment by zbyte64** [comment]
Looks like `graphene_django/tests/test_utils.py` needs to be updated to reference `GraphqlTestCase`? Seeing `ModuleNotFoundError: No module named 'graphene_django.utils.testing.Client'; 'graphene_django.utils.testing' is not a package`

Not sure how master is passing but this does not...

**Comment by ulgens** [comment]
This PR can be closed, in favor of https://github.com/graphql-python/graphene-django/pull/1084

**Pull request closed without merging**